<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DesignerMain</class>
 <widget class="QMainWindow" name="DesignerMain">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1296</width>
    <height>822</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Desinger</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>3</number>
    </property>
    <property name="margin">
     <number>3</number>
    </property>
    <item>
     <widget class="QTabWidget" name="ui_TabWidget">
      <property name="currentIndex">
       <number>-1</number>
      </property>
      <property name="tabsClosable">
       <bool>true</bool>
      </property>
      <property name="movable">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1296</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="newMenuBtn"/>
    <addaction name="openMenuBtn"/>
    <addaction name="saveMenuBtn"/>
    <addaction name="saveAsMenuBtn"/>
    <addaction name="separator"/>
    <addaction name="exitMenuBtn"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>&amp;Edit</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>&amp;View</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuRun">
    <property name="title">
     <string>&amp;Run</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuTool">
    <property name="title">
     <string>&amp;Tool</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>&amp;Help</string>
    </property>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuView"/>
   <addaction name="menuRun"/>
   <addaction name="menuTool"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="saveMenuBtn">
   <property name="text">
    <string>저장(&amp;S)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="newMenuBtn">
   <property name="text">
    <string>새로 만들기(&amp;N)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="openMenuBtn">
   <property name="text">
    <string>열기(&amp;O)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="saveAsMenuBtn">
   <property name="text">
    <string>다른 이름으로 저장(&amp;A)</string>
   </property>
  </action>
  <action name="exitMenuBtn">
   <property name="text">
    <string>끝내기(&amp;X)</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
